config/config.go

```diff
@@ -174,6 +179,7 @@ func (c *Config) Ensure(fromCloud bool, logger logging.Logger) error {
 
 	for idx := 0; idx < len(c.Services); idx++ {
 		service := &c.Services[idx]
+		// dependsOn will only be populated if attributes have been converted.
```

Comment:
Noting this may change based on what you do with the above comment.

Reply:
updated